src/website/views/custom-elements/progress.pug

```diff
@@ -0,0 +1,51 @@
+---
+title: Progress
+scripts:
+  - js/ce/ux-chi-ce
+---
+p.-text Use progress to display the status of a task or process.
+
+h3 Examples
+p.-text
+  | To render a progress element, use the tag <code>&lt;chi-progress&gt;</code>.
+  | Set the <code>value</code> attribute to the amount of work complete. The default state is <code>success</code>.
+
+h4 Base
+.example.-mb--3
+  .-p--3
+    h5 0%
+    chi-progress(value='0')
+    h5 50%
+    chi-progress(value='50')
+    h5 100%
+    chi-progress(value='100')
+  :code(lang="html")
+    <h5>0%</h5>
```

Comment:
I don't think is necessary to include this within code section.

Reply:
@srinath-koneru titles are not necessary here. Custom elements must remain.

Reply:
@SergioQCL @dani-cl-madrid If that's the case won't the user be thinking that just using the ```<chi-progress>``` will also be presenting them with a title. It might get confusing it seems?
The regular component had it so I just went with it.

Reply:
good catch @srinath-koneru! Please let the title.

Reply:
@jllr copy.